procedures/CodeBrowser.ipf

```diff
@@ -878,19 +951,67 @@ Function/S nicifyProcedureList(list)
 	return niceList
 End
 
+// Get the specified line of code from a function
+//
+// @param funcInfo  output of FunctionInfo for the function in question
+// @param lineNo    line number relative to the function definition
+//                  set to -1 to return lines before the procedure that are not part of the preceding macro or function
+//                  see `DisplayHelpTopic("ProcedureText")`
+//
+// @returns lines of code from a function inside a procedure file
+Function/S getFunctionLine(lineNo, funcInfo)
+	variable lineNo
+	string funcInfo
+
+	string funcName, module, procedure, context
+	variable linesOfContext
+
+	funcName = StringByKey("NAME", funcInfo)
+	module = StringByKey("INDEPENDENTMODULE", funcInfo)
+	procedure = StringByKey("PROCWIN", funcInfo)
+
+	if(!isProcGlobal(module))
+		debugPrint(procedure + " is not in ProcGlobal")
+		procedure = procedure + "[" + module + "]"
+	endif
+
+	linesOfContext = lineNo < 0 ? lineNo : 0
+	context = ProcedureText(funcName, linesOfContext, procedure)
+
+	if(lineNo < 0)
+		return context
+	endif
+
+	return StringFromList(lineNo, context, "\r")
+End
+
+Function/S TrimFunctionComment(context)
+	string context
+
+	string re, comment
+
+	// help for regex on https://regex101.com/
+	re="^(?is)(.*)[\r\n]+\s*Function(.*)"
```

Comment:
@t-b need help here. See https://github.com/byte-physics/igor-code-browser/issues/11#issuecomment-445964278

Reply:
Need to exclude lines that are preceding the comment and are not comment lines.